haskell/private/providers.bzl

```diff
@@ -160,7 +160,7 @@ def get_libs_for_ghc_linker(hs, build_info, path_prefix = None):
         library_deps = darwin_convert_to_dylibs(hs, _library_deps)
 
         # GHC produces intermediate dylibs that load the unmangled libraries.
-        ld_library_deps = _ld_library_deps + [
+        ld_library_deps = library_deps + _ld_library_deps + [
```

Comment:
Please add a commend that this duplication is required for GHC 8.4.

Reply:
fixed